test/azure/disk_test.go

```diff
@@ -0,0 +1,55 @@
+// +build azure
+
+// NOTE: We use build tags to differentiate azure testing because we currently do not have azure access setup for
+// CircleCI.
+
+package test
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/azure"
+	"github.com/stretchr/testify/require"
+)
+
+/*
+The below tests are currently stubbed out, with the expectation that they will throw errors.
```

Comment:
Update comment

Reply:
@davesee What was the intention with these comments? Not sure how they should be updated.

Reply:
So the comment references both Azure Virtual Machines and AWS.  need to refactor.